interceptors/Mementifier.cfc

```diff
@@ -94,8 +94,11 @@ component{
 		excludes="",
 		struct mappers={},
 		struct defaults={},
-		boolean ignoreDefaults=false
+        boolean ignoreDefaults=false,
+        boolean trustedGetters
 	){
+        param arguments.trustedGetters = $mementifierSettings.trustedGetters;
```

Comment:
What happens if you do this: `boolean trustedGetters = $mementifierSettings.trustedGetters` in the argument definition?

Reply:
Good point.